docs/api/migration-guides/qiskit-runtime-setup.mdx

```diff
@@ -0,0 +1,301 @@
+| qiskit.providers.ibmq.runtime.RuntimeEncoder     | qiskit_ibm_runtime.RuntimeEncoder         |
+| qiskit.providers.ibmq.runtime.RuntimeDecoder     | qiskit_ibm_runtime.RuntimeDecoder         |
+| qiskit.providers.ibmq.runtime.ParameterNamespace |                                           | This class was used for custom programs, which are no longer supported.
+| qiskit.providers.ibmq.runtime.RuntimeOptions     | qiskit_ibm_runtime.RuntimeOptions         |
+
+
+## Import path
+
+The import path has changed as follows:
+
+<Tabs>
+  <TabItem value="updated" label="Updated">
+``` python
+from qiskit_ibm_runtime import QiskitRuntimeService
+```
+  </TabItem>
+
+  <TabItem value="legacy" label="Legacy">
+``` python
+from qiskit import IBMQ
+```
+  </TabItem>
+</Tabs>
+
+## Save accounts
+
+Use the updated code to work save accounts.
+
+<Tabs>
+  <TabItem value="updated" label="Updated">
+The new syntax accepts credentials for
+Qiskit Runtime on IBM Cloud or IBM Quantum Platform. For more
+information on retrieving account credentials, see [Install and set up](../../start/install).
+
+``` python
+# IBM Cloud channel
+
+QiskitRuntimeService.save_account(channel="ibm_cloud", token="<IBM Cloud API key>", instance="<IBM Cloud CRN>", overwrite=True)
+
+# IBM Quantum channel; set to default 
+
+QiskitRuntimeService.save_account(channel="ibm_quantum", token="<IQP_TOKEN>", overwrite=True, default=true)
+```
+
+Additionally, you can now name your saved credentials and load the credentials by name.
+
+
+``` python
+# Save different accounts for open and premium access
+
+QiskitRuntimeService.save_account(channel="ibm_quantum", token="<IQX_TOKEN>", instance="h1/g1/p1", name="premium")
+QiskitRuntimeService.save_account(channel="ibm_quantum", token="<IQX_TOKEN>", instance="h2/g2/p2", name="open")
+
+# Load the "open" credentials
+
+service = QiskitRuntimeService(name="open")
+```
+  </TabItem>
+
+  <TabItem value="legacy" label="Legacy">
+``` python
+IBMQ.save_account("<IQX_TOKEN>", overwrite=True)
+```
+  </TabItem>
+</Tabs>
+
+
+## Load accounts
+
+Use the updated code to load accounts.
+
+<Tabs>
+  <TabItem value="updated" label="Updated">
+The new syntax combines the functionality from `load_account()` and
+`get_provider()` in one statement. The `channel` input parameter is
+optional. If multiple accounts have been saved in one device and no
+`channel` is provided, the default is `"ibm_cloud"`.
+
+``` python
+# To access saved credentials for the IBM cloud channel
+service = QiskitRuntimeService(channel="ibm_cloud")
+
+# To access saved credentials for the IBM quantum channel
+service = QiskitRuntimeService(channel="ibm_quantum")
+```
+  </TabItem>
+
+  <TabItem value="legacy" label="Legacy">
+``` python
+IBMQ.load_account()
+```
+  </TabItem>
+</Tabs>
+
+
+## Channel selection (get a provider)
+
+Use the updated code to select a channel.
```

Comment:
I think this is meant to show selecting an instance (formerly known as provider)

```suggestion
## Instance selection (get a provider)

Use the updated code to select an instance.
```